website/src/documentation/transforms/python/element-wise/withtimestamps.md

```diff
@@ -19,10 +19,116 @@ limitations under the License.
 -->
 
 # WithTimestamps
+
+<script type="text/javascript">
+localStorage.setItem('language', 'language-py')
+</script>
+
 Assigns timestamps to all the elements of a collection.
 
 ## Examples
-See [BEAM-7389](https://issues.apache.org/jira/browse/BEAM-7389) for updates. 
 
-## Related transforms 
-* [Reify]({{ site.baseurl }}/documentation/transforms/python/elementwise/reify) converts between explicit and implicit forms of Beam values.
\ No newline at end of file
+In the following examples, we create a pipeline with a `PCollection` and attach a timestamp value to each of its elements.
+Timestamps are especially useful on streaming pipelines where windowing and late data play a more important role.
```

Comment:
Clause order

"When windowing and late data play an important role in streaming pipelines, timestamps are especially useful."

Reply:
Thanks, reworded